Makefile

```diff
@@ -14,22 +14,64 @@ PKG=github.com/hashicorp/vault-k8s/version
 LDFLAGS?="-X '$(PKG).Version=v$(VERSION)'"
 TESTARGS ?= '-test.v'
 
-.PHONY: all test build image clean version
+HELM_CHART_VERSION ?= 0.25.0
+
+.PHONY: all test build image clean version deploy exercise teardown
 all: build
 
 version:
 	@echo $(VERSION)
 
 build:
 	CGO_ENABLED=0 GOOS=$(GOOS) GOARCH=$(GOARCH) go build \
-		-a \
```

Comment:
I'm curious about what motivated this change.

Reply:
With all the k8s dependencies, the injector takes a while to rebuild from scratch, but I'm confident in Go's build caching correctness so I wanted to use the cache for speed. If we have any concerns about using the build cache though, we should probably disable the cache from the setup-go step in the CRT build so that it's still doing a build from scratch everytime.